Cargo.toml

```diff
@@ -22,7 +22,7 @@ log = "0.4.11"
 notify = { version = "4.0.12", optional = true }
 
 # async
-tokio = { version = "1.0", features = ["rt"], optional = true }
+tokio = { version = "1.0", features = ["rt", "time"], optional = true }
```

Comment:
Why do we need `tokio/time` if only `tokio/rt` is explicitly used?

Reply:
Let me rephrase: I see `tokio::time` is used in the async example, but why is the `async` feature only depending on `tokio/rt`? Don't we need at least a dev dependency?

Reply:
Nevermind I just saw the example dependency. That GitHub UI on phone... 😂